src/EnergyPlus/Humidifiers.cc

```diff
@@ -728,7 +728,7 @@ namespace Humidifiers {
 			}
 
 			if ( !HardSizeNoDesRun ) {
-				NomCapVolDes = MassFlowDes * ( OutletHumRatDes - InletHumRatDes ) / AirDensity;
+				NomCapVolDes = MassFlowDes * ( OutletHumRatDes - InletHumRatDes ) / RhoH2O( InitConvTemp );

```

Comment:
Added autosizing unit test for gas humidifier. Also cleanedup commented out code lines.